# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/man/hotfix-icov_t.patch.xtensa
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- man-db-2.13.0/src/manconv.c.vanilla	2025-04-22 03:12:41.242011117 +0100
+++ man-db-2.13.0/src/manconv.c	2025-04-22 03:17:45.001877085 +0100
@@ -298,7 +298,7 @@
 	const char *input;
 	static char *utf8 = NULL, *output = NULL;
 	size_t utf8left = 0;
-	iconv_t cd_utf8, cd = NULL;
+	iconv_t cd_utf8, cd = (iconv_t)(-1);
 	bool to_utf8 = STREQ (try_to_code, "UTF-8") ||
 	               STRNEQ (try_to_code, "UTF-8//", 7);
 	const char *utf8_target = last ? "UTF-8//IGNORE" : "UTF-8";
